README.md

```diff
@@ -1,60 +1,88 @@
 CakePHP Sample App on OpenShift
------------------
+===============================
 
-This example will serve an http response of various "machine" stats from the "machine" the CakePHP app is running on to [http://host:8080](http://host:8080).
+This is a quickstart CakePHP application for OpenShift v3 that you can use as a starting point to develop your own application and deploy it on an [OpenShift](https://github.com/openshift/origin) cluster.
 
-### OpenShift setup ###
+If you'd like to install it, follow [these directions](https://github.com/openshift/cakephp-ex/blob/master/README.md#installation).  
 
-One possible option is to use the Docker all-in-one launch as described in the [origins project](https://github.com/openshift/origins).
+The steps in this document assume that you have access to an OpenShift deployment that you can deploy applications on.
 
-### The project ###
+OpenShift Considerations
+------------------------
+These are some special considerations you may need to keep in mind when running your application on OpenShift.
 
-If you don't have a project setup all ready, go ahead and take care of that
+###Security
```

Comment:
What about my previous comment about this section ? Or it doesn't make sense ? :-P
_______
I would get rid ofthe first two paragraphs in the 'Security' section and replace it with:

Since the quickstarts are shared code, we had to take special consideration to ensure that security related configuration variables was unique across applications. To accomplish this, we modified some of the configuration files. Namely we changed Security.salt and Security.cipherSeed values in the app/Config/core.php config file. Those values are now generated from the application template as CAKEPHP_SECURITY_SALT and CAKEPHP_SECURITY_CIPHER_SEED. Also the secret token is generated in the template as CAKEPHP_SECRET_TOKEN. From these values the session hashes for are generated. Now instead of using the same default values, OpenShift can generate these values using the generate from logic defined within the instant application's template.

Thoughts ?